CodeForBaltimore/Bmore-Responsive
# Fix Bugs with CSV Exports

- The csv contents were not being correctly attached to the api response object
 - Entity addresses are stored as JSON Objects and needed to be formatted so that they look correct in the export
 - format contact's phone and email values for csv exports

# Description
https://github.com/CodeForBaltimore/Healthcare-Rollcall/issues/108
I found a frontend issue on Healthcare Rollcall that was related to CSV exports.  When I went to go take care of that feature, I noticed that there was already an "Export to CSV" button on the front-end.  But when I clicked it, the only thing that happened was a csv was downloaded that contained the word "OK".  I fixed that, but then saw that the address field needed some extra formatting, so I took care of that too.

## Related PRs
N/A

## Todos


## Deploy Notes

Notes regarding deployment the contained body of work. 

## Steps to Test or Reproduce

Log into Healthcare Rollcall
Click "Export"
Look at the csv that's generated to make sure it matches the list of facilities that are shown on the front-end

## Impacted Areas in Application

CSV Exports of facilities and contacts
